### PR TITLE
Separate css styles

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]
Please break this into multiple PRs. Create this to be more component based so it can be merged and verified easier

File: src/components/PositionCard/PositionCard.tsx [hunk 1]
```diff
@@ -78,220 +63,54 @@ export const MinimalPositionCard: React.FC<PositionCardProps> = ({
...
-          pair.getLiquidityValue(
-            pair.token1,
-            totalPoolTokens,
-            userPoolBalance,
-            false,
-          ),
-        ]
-      : [undefined, undefined];
-
-  return (
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/PositionCard/PositionCard.tsx:192]: I removed the unused components

File: src/components/AddressInput/AddressInput.tsx [hunk 2]
```diff
@@ -47,12 +21,15 @@ const AddressInput: React.FC<AddressInputProps> = ({
...
   const error = Boolean(value.length > 0 && !loading && !address);
-  const classes = useStyles({ error });
 
   return (
-    <Box className={classes.addressInput}>
-      <Box display='flex' justifyContent='space-between' alignItems='center'>
-        <Typography>{label}</Typography>
+    <Box
+      className={`addressInput ${
+        error ? 'border-error' : 'border-primaryDark'
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/AddressInput/AddressInput.tsx:28]: It was passing error to the styles hook but I removed the styles hook and added class based on error

File: src/components/AddressInput/AddressInput.tsx [hunk 3]
```diff
@@ -65,6 +42,7 @@ const AddressInput: React.FC<AddressInputProps> = ({
       </Box>
       <input
         value={value}
+        className={error ? 'text-error' : 'text-primaryText'}
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/AddressInput/AddressInput.tsx:45]: This is same as above

File: src/components/PairsTable/PairsTable.tsx [hunk 4]
```diff
@@ -108,68 +108,37 @@ const PairTable: React.FC<TokensTableProps> = ({ data }) => {
...
-          alignItems='center'
-          justifyContent='space-between'
-        >
-          <Typography variant='body1'>24h Fees</Typography>
-          <Typography variant='body1'>
-            ${Number(oneDayFee).toLocaleString()}
-          </Typography>
+        <Box className='mobileRow'>
+          <p>24h Fees</p>
+          <p>${oneDayFee}</p>
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/PairsTable/PairsTable.tsx:141]: It was showing `24h Fees` as `NaN` on mobile view and this fixed the issue.

File: src/components/StakeQuickModal/StakeQuickModal.tsx [hunk 5]
```diff
@@ -172,7 +143,7 @@ const StakeQuickModal: React.FC<StakeQuickModalProps> = ({ open, onClose }) => {
                 max={100}
                 step={1}
                 value={stakePercent}
-                onChange={(evt: any, value) => {
+                handleChange={(evt: any, value) => {
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/StakeQuickModal/StakeQuickModal.tsx:142]: It returned error after moving CSS of `ColoredSlider` component and updated `onChange` prop to `handleChange` to fix the error.

File: src/components/StakeSyrupModal/StakeSyrupModal.tsx [hunk 6]
```diff
@@ -243,16 +207,14 @@ const StakeSyrupModal: React.FC<StakeSyrupModalProps> = ({
                 max={100}
                 step={1}
                 value={stakePercent}
-                onChange={(_, value) => {
+                handleChange={(_, value) => {
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/StakeSyrupModal/StakeSyrupModal.tsx:210]: It returned error after moving CSS of `ColoredSlider` component and updated `onChange` prop to `handleChange` to fix the error.

File: src/components/SwapTokenDetails/SwapTokenDetails.tsx [hunk 7]
```diff
@@ -155,39 +130,28 @@ const SwapTokenDetails: React.FC<{
...
-      <Box
-        display='flex'
-        justifyContent='space-between'
-        alignItems='center'
-        py={1}
-        px={2}
-      >
+      <Box className='flex justify-between items-center' py={1} px={2}>
         <a
           href={`https://polygonscan.com/token/${tokenAddress}`}
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/SwapTokenDetails/SwapTokenDetails.tsx:145]: We are not using testnet for now and I will update this in another PR.

File: src/components/ButtonSwitch/ButtonSwitch.tsx [hunk 8]
```diff
@@ -133,12 +55,14 @@ const ButtonSwitch: React.FC<ButtonSwitchProps> = ({
...
-                  value === tab.value ? classes.tabActive : classes.tabInactive,
-                  't-button-switch-item',
-                )}
-                style={{ minWidth: '0px' }}
+                className={`tab ${
+                  value === tab.value ? 'tabActive' : 'tabInactive'
+                }`}
+                style={{
+                  minWidth: '0px',
+                  minHeight: minHeight,
```
[the remark this answers is not in the document]
Comment (totop716) [reply src/components/ButtonSwitch/ButtonSwitch.tsx:62]: This is to fix `ButtonSwitch` on the `Info` section of swap pro mode

File: yarn.lock [hunk 9]
```diff
@@ -12352,6 +12352,11 @@ immer@8.0.1, immer@^9.0.6, immer@^9.0.7:
   resolved "https://registry.yarnpkg.com/immer/-/immer-9.0.12.tgz#2d33ddf3ee1d247deab9d707ca472c8c942a0f20"
   integrity sha512-lk7UNmSbAukB5B6dh9fnh5D0bJTOFKxVg2cyJWTYrWRfhLrLMBquONcUs3aFq507hNoIZEDDh8lb8UtOizSMhA==
 
+immutable@^4.0.0:
```
Comment (FriskyHamTitz) [inline yarn.lock:12355]: What is this used for why don't I see any add to package.json for this?

**Comment by FriskyHamTitz** [comment]
Please resolve all the conflicts. After we merge this I'm going to create a new branch for the next version that we can start working on.

**Pull request merged**